cmake/TestSetup.cmake

```diff
@@ -109,10 +113,13 @@ function(setup_test_mpi nproc dir arg extra_labels)
     list(APPEND labels "regression;mpi-yes")
     list(APPEND labels "${extra_labels}")
 
-    if( "${Kokkos_ENABLE_CUDA}" )
+    if(Kokkos_ENABLE_CUDA)
       set(PARTHENON_KOKKOS_TEST_ARGS "--kokkos-num-devices=${NUM_GPU_DEVICES_PER_NODE}")
       list(APPEND labels "cuda")
     endif()
+    if (Kokkos_ENABLE_OPENMP)
+      set(PARTHENON_KOKKOS_TEST_ARGS "${PARTHENON_KOKKOS_TEST_ARGS} --kokkos-threads=${NUM_OMP_THREADS_PER_RANK}")
+    endif()
```

Comment:
If Cuda and OpenMP are enabled, aren't the options overridden?

Reply:
Do you mean `--kokkos-threads` would override `--kokkos-num-devices`? The second set command appends --kokkos-threads to the previously contents of `PARTHENON_KOKKOS_TEST_ARGS`. See:
```cmake
set(PARTHENON_KOKKOS_TEST_ARGS "${PARTHENON_KOKKOS_TEST_ARGS}
```